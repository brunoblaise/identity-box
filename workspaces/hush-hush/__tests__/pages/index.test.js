import React from 'react'
import { render } from '@testing-library/react'
import Home from '../../pages/index'

describe('index', () => {
  beforeEach(() => {
    process.env.serviceUrl = {
      test: 'http://localhost:3000'
    }
  })

  it('has proper styling', () => {
    const { container } = render(<Home />)
    expect(container).toMatchSnapshot()
  })

  it('displays welcome text', () => {
    const { getByText } = render(<Home />)
    expect(getByText(/Hush Hush needs to know two things/i)).toBeInTheDocument()
  })
})
