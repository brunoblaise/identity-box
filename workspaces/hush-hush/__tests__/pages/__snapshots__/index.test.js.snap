// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index has proper styling 1`] = `
.emotion-4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-flow: row nowrap;
  -ms-flex-flow: row nowrap;
  flex-flow: row nowrap;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  height: 100vh;
  width: 100vw;
}

.emotion-2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-flow: column nowrap;
  -ms-flex-flow: column nowrap;
  flex-flow: column nowrap;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  width: 100%;
  opacity: 0;
}

.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-flow: column nowrap;
  -ms-flex-flow: column nowrap;
  flex-flow: column nowrap;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

<div>
  <div
    class="emotion-4 emotion-5"
  >
    <div
      css="[object Object]"
    >
      <div
        class="emotion-2 emotion-3"
        opacity="0"
      >
        <div
          class="emotion-0 emotion-1"
        >
          <div
            css="[object Object]"
          >
            In order to share your secret securely with the intendent (and only intended) recipient, Hush Hush needs to know two things:
          </div>
          <div
            css="[object Object]"
          >
            1. The DID of the intended recipient.
          </div>
          <div
            css="[object Object]"
          >
            2. Your current DID.
          </div>
          <div
            css="[object Object]"
          >
            For this reason, Hush Hush, needs to connect to your mobile where you can then select the intended recipient from your 
            <span
              css="[object Object]"
            >
              Identity Box IdApp Address Book
            </span>
            .
          </div>
          <div
            css="[object Object]"
          />
          <div
            css="[object Object]"
          >
            Please make sure that you have your 
            <span
              css="[object Object]"
            >
              IdApp
            </span>
             ready and that you have selected the idenity you want to use before hitting 
            <span
              css="[object Object]"
            >
              connect
            </span>
             below.
          </div>
          <div
            css="[object Object]"
          />
          <div
            css="[object Object]"
          >
            <button
              class="ui primary button"
            >
              Connect...
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
